### DISCUSSION TITLE
Not a single next CLI command works! (was working fine yesterday)

### DISCUSSION THREAD

**Discussion by Roaldkamman** [vercel/next.js#21968] [Help]
**What version of Next.js are you using?**

I can't even check

**What version of Node.js are you using?**

v12.19.0

**What browser are you using?**

NA

**What operating system are you using?**

Windows

**How are you deploying your application?**

NA

**Describe the Bug**

npm run dev command worked fine yesterday. 
Today nothing that has anything to do with next.js works in the CLI. 
Even npx next -h is giving me the exact same error as running npm run dev. 

Instead this is the response regardless of the command: 
```
rjkam@LAPTOP-FLVMKRGQ MINGW64 /c/projects/shopifytestapp (main)
$ npx next -h
internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module 'chalk'
Require stack:
- C:\projects\shopifytestapp\node_modules\next\dist\build\output\log.js
- C:\projects\shopifytestapp\node_modules\next\dist\bin\next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (C:\projects\shopifytestapp\node_modules\next\dist\build\output\log.js:1:210)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:903:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\projects\\shopifytestapp\\node_modules\\next\\dist\\build\\output\\log.js',
    'C:\\projects\\shopifytestapp\\node_modules\\next\\dist\\bin\\next'
  ]
}
npm ERR! code 1
npm ERR! path C:\projects\shopifytestapp
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c next -h

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\rjkam\AppData\Local\npm-cache\_logs\2021-02-07T21_22_22_549Z-debug.log
```

**Expected Behavior**

The server to start or list of commands. (Both throw the same error) 

**To Reproduce**

Use literally any next.js CLI command

**Comment by Roaldkamman** [comment]
I redid a manual install of next and made sure that the right scripts are in place as per: https://nextjs.org/docs
Problem persists.

**Comment by Roaldkamman** [comment]
created a new next app to test and the commands work there. Strange since I literally didn't change anything in the current app yet.  Will work in the working app in the meantime. Hope it doesn't stop working like this one...

**Comment by flppv** [comment]
+1. For me it doesn't see webpack
```
Error: Cannot find module 'webpack'
Require stack:```

**Comment by tarunama** [comment]
@Roaldkamman 
You could be better to add info to issue template.

**Reply by Roaldkamman** [reply]
Which Info are you looking for specifically? Here is the code. https://github.com/Roaldkamman/shopifytestapp

**Comment by Roaldkamman** [comment]
The only thing I can think of is that something went wrong during create-next-app. I just have no idea why it would work one day and flatline the next when the code is identical? Just tested if errors persist today and they do... Luckily my new work environment does work. If this happens to you I recommend just creating a new next app. And copy changes you made in your app to the new environment.
